tripy/tests/integration/test_dequantize.py

```diff
@@ -29,6 +29,7 @@ class TestDequantize:
     @pytest.mark.parametrize(
         "dtype", [tp.float32, tp.float16, pytest.param(tp.bfloat16, marks=skip_if_older_than_sm80)]
     )
+    @pytest.mark.skip("StableHLO QDQ broken")
```

Comment:
Can we xfail these instead of skip? That way we will remember to turn them on again when we switch to TRT dialect.